niworkflows/utils/bids.py

```diff
@@ -41,7 +41,7 @@
     'pet': {'suffix': 'pet'},
     'roi': {'datatype': 'anat', 'suffix': 'roi'},
     'sbref': {'datatype': 'func', 'suffix': 'sbref', 'part': ['mag', None]},
-    't1w': {'datatype': 'anat', 'suffix': 'T1w', 'part': ['mag', None]},
+    't1w': {'suffix': 'T1w'},
```

Comment:
rather than change the default queries, you can pass in alternative queries by using `collect_data(..., queries=x)`

Reply:
This should be reverted